ai/README.md

```diff
@@ -0,0 +1,155 @@
+[![gdxAI](https://img.shields.io/badge/gdxAI-1.8.2-red.svg)](https://github.com/libgdx/gdx-ai)
```

Comment:
May I ask you to also update `.github/CONTRIBUTING.md` and include gdxAI in the _Updating dependencies_ section?

Reply:
Sure, no problem. I also added a link to the gdxAI repo at the bottom of ai/README. I skipped alternatives to gdxAI, because I don't know of any meaningful Kotlin game AI library.